scripts/peering-prefix

```diff
@@ -171,17 +188,23 @@ c|C)
     if [[ $ctoken -gt 65535 || $ctoken -lt 0 ]] ; then
         die "error [-c ASN:COMM accepts only 16-bit numbers]"
     fi
-    communities+=($OPTARG)
+    communities+=("$OPTARG")
     ;;
 l)
-    if ! [[ $OPTARG =~ [0-9]{1,9},[0-9]{1,9},[0-9]{1,9} ]] ; then
+    if ! [[ "$OPTARG" =~ [0-9]{1,9},[0-9]{1,9},[0-9]{1,9} ]] ; then
         die "error [-l requires a string that matches '[0-9]{1,9},[0-9]{1,9},[0-9]{1,9}']"
     fi
-    large_communities+=($OPTARG)
+    large_communities+=("$OPTARG")
     ;;
 R)
     config_dataplane=0
     ;;
+M)
+    if [ ! -e "$ctxid_db" ] ; then
+        die "error [-M requires $ctxid_db]"
```

Comment:
Let's make this a bit more verbose, like "error [create a $ctxid_db file with your container's ID]"